lib/bag/item.rb

```diff
@@ -23,6 +23,22 @@ def item_id
     object_properties['objectId'].first.tr('/', '-')
   end
 
+  def owner_id
+    object_properties['ownerId']
+  end
+
+  def other_ids
```

Comment:
the nature of `#split` is that it returns an array regardless of if it finds the string that it is splitting on. 

```
"blahblah".split(',') #returns ["blahblah"]
```

So this method can be simplified to be just:
`object_properties['otherIds'].split(',')`